### DISCUSSION TITLE
Mac OS Support Command-M hotkey to minimize

### DISCUSSION THREAD

**Discussion by carsongregory** [dbeaver/dbeaver#11657] [Q&A] [answered]
Requesting to please add support for hotkey `Command-M` to minimize DBeaver application window on Mac OS.

This is a convention followed by most apps.

Thank you!
Carson

**Comment by kseniiaguzeeva** [comment, marked as answer]
You can set custom hotkeys in Window>Preferences>User interface>Keys
![Screenshot 2021-03-09 at 13 02 22](https://user-images.githubusercontent.com/51405061/110454007-d54f4c80-80d7-11eb-9a31-f30b0ff26677.png)
